<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepthMapViewOptions</class>
 <widget class="QWidget" name="DepthMapViewOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="color">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="depth">
       <property name="text">
        <string>Depth</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="depthOptions">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>16</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="weight">
       <property name="text">
        <string>Weight</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="weightOptions">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>16</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="uncertainty">
       <property name="text">
        <string>Uncertainty</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="uncertaintyOptions">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>16</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>depth</sender>
   <signal>toggled(bool)</signal>
   <receiver>depthOptions</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weight</sender>
   <signal>toggled(bool)</signal>
   <receiver>weightOptions</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uncertainty</sender>
   <signal>toggled(bool)</signal>
   <receiver>uncertaintyOptions</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
